src/libs/actions/App.js

```diff
@@ -206,6 +215,39 @@ function setUpPoliciesAndNavigate(session) {
         });
 }
 
+function openProfile() {
+    const oldTimezoneData = myPersonalDetails.timezone || {};
+    const newTimezoneData = {
+        automatic: lodashGet(oldTimezoneData, 'automatic', true),
+        selected: moment.tz.guess(true),
+    };
```

Comment:
@Beamanator Ok, so you will pass in the new one even if we do not want it automatically updated and in that case this just wont be saved in Web-E?

Reply:
yeah exactly